src/cmd/build.js

```diff
@@ -31,8 +31,9 @@ function defaultPackageCreator(
           filter: (...args) => fileFilter.wantFile(...args),
         })
         .then((buffer) => {
+          let extension = 'zip';
           let packageName = safeFileName(
-            `${manifestData.name}-${manifestData.version}.xpi`);
+            `${manifestData.name}-${manifestData.version}.${extension}`);
```

Comment:
Oh yeah, you’re right. I was thinking of making the output extension an
option but then realised it’s unneeded. Shall remove.